src/org/labkey/remoteapi/issues/IssueCommand.java

```diff
@@ -0,0 +1,68 @@
+package org.labkey.remoteapi.issues;
+
+import org.apache.http.client.methods.HttpPost;
+import org.apache.http.client.methods.HttpUriRequest;
+import org.apache.http.entity.ContentType;
+import org.apache.http.entity.mime.HttpMultipartMode;
+import org.apache.http.entity.mime.MultipartEntityBuilder;
+import org.json.JSONArray;
+import org.json.simple.JSONObject;
+import org.labkey.remoteapi.PostCommand;
+
+import java.io.File;
+import java.net.URI;
+
+public class IssueCommand extends PostCommand<IssueResponse>
```

Comment:
Command name should match the action name.
```suggestion
public class IssuesCommand extends PostCommand<IssueResponse>
```

Reply:
done